test/index.test.js

```diff
@@ -17,7 +17,7 @@ governing permissions and limitations under the License.
 
 const Swagger = require('swagger-client')
 const sdk = require('../src')
-const fetch = require('cross-fetch')
```

Comment:
cross-fetch is still used in some cases right?

Reply:
Cross-fetch is the swagger-default module and it's also in the mocks for the tests so I didn't want to mess with it.  Technically people can revert to the old behavior by providing "useSwaggerFetch: true" to the options.